graphql/schema.graphql

```diff
@@ -41,14 +41,20 @@ type Patch {
   author: String!
   version: String!
   status: String!
-  createTime: Time!
-  startTime: Time
-  finishTime: Time
+  createTime: String!
```

Comment:
These fields are now incompatible with the base model. What was the reason for changing them?

Reply:
A gql should not reflect the DB or the models of the backend. It should reflect the needs of the UI and these are the needs of the UI.

Reply:
What are those needs? 

I don't think that the backend should be responsible for string formatting dates (makes timezones difficult to implement if nothing else.)

Reply:
I'm doing two PRs right now that were combined but then i separated them. Changing these fields is supposed to be in the NEXT pr, not this one. I will change these fields back and in [this pr](https://github.com/evergreen-ci/evergreen/pull/3167) you will see why those fields are changed.

Reply:
Right, my point was that this change makes it so that gqlgen can no longer generate these fields, and you didn't add a resolver

Reply:
> What are those needs?
> 
> I don't think that the backend should be responsible for string formatting dates (makes timezones difficult to implement if nothing else.)

the needs are that the date is formatted. the timezone is not difficult to implement since it's stored in the user's settings.

Reply:
PLEASE IGNORE THE CODE RELATED TO THESE FIELDS IN THIS PR

Reply:
right, but once you turn the date into a string, to present a timezone different from what the server gives you.

Reply:
that's true but i'm not sure it's a real issue if all our users have their time zones saved in their settings